ci/licenses.sh

```diff
@@ -10,7 +10,7 @@ exitStatus=0
 dart --version
 
 # These files trip up the script on Mac OS X.
-find . -name ".DS_Store" | xargs rm
+find . -name ".DS_Store" -exec rm {} \;
```

Comment:
uber-nit: you can just do `-delete`.

Reply:
right on, i would change it but end of day and ci takes forever

Reply:
Don't you need quotes to deal with paths with spaces here?